docs/samples/Microsoft.ML.Samples/Dynamic/PermutationFeatureImportance/PfiBinaryClassificationExample.cs

```diff
@@ -66,7 +66,7 @@ public static void Example()
             // Third, some features show an *increase* in AUC. This means that the model actually improved 
```

Comment:
There's a few usages of `AUC` here and in the `Console.Writeline` below.

Reply:
During my original work, I did consider this but decided against this. So, the reason why we changed it at all is just to be consistent with .NET member naming guidelines, not because we had any illusions that anyone would be genuinely confused about what AUC actually meant. I think it's fine and desirable for samples to continue to call this metric by its canonical, accepted name in anything meant to be read and interpreted by a human. [Certainly elsewhere people don't have too many qualms about this.](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.auc.html#sklearn.metrics.auc)